src/components/PrizePage/Prizes.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import Container from '@material-ui/core/Container';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import MuiTableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import useTheme from '@material-ui/core/styles/useTheme';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import DesignHack from './images/design-hack.png';
+import GameHack from './images/game-hack.png';
+import MLHack from './images/ml-hack.png';
+import MobileHack from './images/mobile-hack.png';
+import OverallHack from './images/overall-hack.png';
+import WebHack from './images/web-hack.png';
```

Comment:
I got these off of the Figma, but perhaps we can design our own icons for the prize tracks? Or look for higher quality images?